app/scripts/models/auth_brokers/web-channel.js

```diff
@@ -54,6 +54,55 @@ define([
       return p();
     },
 
+    // WebChannel reliers can request access to relier-specific encryption
+    // keys.  In the future this logic may be lifted into the base OAuth class
+    // and made available to all reliers, but we're putting it in this subclass
+    // for now to guard against accidental exposure.
+
+    getOAuthResult: function (account) {
+      var self = this;
+      return OAuthAuthenticationBroker.prototype.getOAuthResult.call(this, account)
+        .then(function (result) {
+          if (! self.relier.isKeyFetchEnabled()) {
+            return result;
+          }
+          return self.deriveRelierKeys(account)
+            .then(function (keys) {
+              result.keys = keys;
+              return result;
+            });
+        });
+    },
+
+    deriveRelierKeys: function (account) {
+      var self = this;
+      // XXX TODO: pass an fxaClient around properly
+      var fxaClient = this._assertionLibrary._fxaClient;
+      var masterKeys;
+      var relierKeys = {};
+      var infoPrefix = 'identity.mozilla.com/picl/v1/oauth/';
```

Comment:
Can you move this prefix to [constants.js](https://github.com/mozilla/fxa-content-server/blob/14cbf41d78d918382fc414d67abc0a4fa6d81619/app/scripts/lib/constants.js)?